test/parse_utils_test.ts

```diff
@@ -3,7 +3,7 @@ import { expect } from '@0x/contracts-test-utils';
 import { NULL_ADDRESS } from '@0x/utils';
 import 'mocha';
 
-import { ApiKeyStructure, getApiKeyWhitelistWithFallback } from '../src/config';
+import { ApiKeyStructure, getApiKeyWhitelist } from '../src/config';
 import { parseUtils } from '../src/utils/parse_utils';
 
 const SUITE_NAME = 'parseUtils';
```

Comment:
can delete line `delete process.env.TEST_LEGACY_KEY;` below this

Reply:
good call